util/util.c

```diff
@@ -280,3 +280,26 @@ uint32_t srt_to_uint32(const char *str)
 	else
 		return value;
 }
+
+void put_unaligned_be16(uint8_t *buf, uint16_t val)
```

Comment:
I think the whole point of these macros is to be able to fully (or almost fully) copy paste some of our linux driver code to no-os driver code - so 1st parameter should be val, 2nd parameter should be buf

Reply:
good point